### DISCUSSION TITLE
couldnot launch instance

### DISCUSSION THREAD

**Discussion by prasannauma** [learndevopsonline/learndevopsonline#97] [Q&A] [answered]
![Screenshot (314)](https://user-images.githubusercontent.com/121955508/220090299-1762d15e-a571-4eb8-adc2-a2c385884bb8.png)

**Comment by learninguser** [comment, marked as answer]
Hi, Please ensure that all the unnecessary tags such as Elastic graphics is not selected under Resource types when creating an instance

**Reply by prasannauma** [reply]
thanks pavan got it

**Reply by learninguser** [reply]
Hi, thanks for letting me know. Please mark this query as answered.
